packages/snaps-sdk/src/types/methods/get-preferences.ts

```diff
@@ -13,9 +13,23 @@ export type GetPreferencesParams = never;
  * @property locale - The user's selected locale.
  * @property currency - The user's selected currency.
  * @property hideBalances - Whether the user has chosen to hide balances.
+ * @property useSecurityAlerts - Whether to run transactions and signatures through blockaid.
+ * @property useSimulations - Whether to simulate transactions and signatures.
+ * @property useTokenDetection - Whether to auto-detect tokens.
+ * @property batchCheckBalances - Whether to fetch balances in an aggregated manner.
+ * @property displayNftMedia - Whether to display NFT media.
+ * @property useNftDetection - Whether to auto-detect NFTs.
+ * @property usePriceCheck - Whether to get token price data from CoinGecko and CryptoCompare.
```

Comment:
Snaps can use any source to get the price, so I'd change this to be a bit more generic.

Reply:
True, i'll change it.